bucket/bomi.json

```
@@ -0,0 +1,35 @@
+{
+    "homepage": "https://bomi-player.github.io/",
+    "license": "GPL-2.0-only",
+    "version": "0.9.11",
```

```suggestion
    "version": "0.9.11",
    "description": "A powerful and easy-to-use multimedia player, formerly known as CMPlayer.",
    "homepage": "https://bomi-player.github.io/",
    "license": "GPL-2.0-only",
```
Put the items in the order of `version`, `description`, `homepage`, `license`.
Also, you `description` is missing.